beacon-chain/db/kv/blocks.go

```diff
@@ -349,7 +355,13 @@ func (k *Store) SaveGenesisBlockRoot(ctx context.Context, blockRoot [32]byte) er
 // fetchBlockRootsBySlotRange looks into a boltDB bucket and performs a binary search
 // range scan using sorted left-padded byte keys using a start slot and an end slot.
 // If both the start and end slot are the same, and are 0, the function returns nil.
-func fetchBlockRootsBySlotRange(bkt *bolt.Bucket, startSlotEncoded, endSlotEncoded interface{}) [][]byte {
+func fetchBlockRootsBySlotRange(
+	bkt *bolt.Bucket,
+	startSlotEncoded interface{},
```

Comment:
Why is it an `interface{}`? Shouldn't it be uint64?

Reply:
It comes from the filters object, which is Enum->interface{}, and then I assert. I wanted to keep that logic within this function as its pretty self-contained